docs/usecases/enable_huge_pages.md

```diff
@@ -0,0 +1,11 @@
+# ENABLE HUGE PAGES
+
+If you need to enable Huge Pages please use this config file:
+
+`ocs_ci/templates/ocp-deployment/huge_pages.yaml`
+
+If you pass this file via --ocsci-conf, it will apply this file:
+
+`ocs_ci/templates/ocp-deployment/huge_pages.yaml`
```

Comment:
we need to enable huge pages using conf/ocsci/enable_huge_pages.yaml

Reply:
Done, yep, I copied wrong file  :( the same in verification